src/server/src/entities/documents.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    ManyToOne,
+    BaseEntity,
+} from 'typeorm';
+import { Student } from '../entities/student';
```

Comment:
I think you can just do ./student to get the Student class

Reply:
that gave me an error though, saying Student class has no default export. Do I add the export to Student class?